src/components/Map/markerStyle.tsx

```diff
@@ -32,13 +34,19 @@ export function createSensorIcon(
 
   // Set marker size
   const standardMarkerSize = 20;
-  const hoverMarkerSize = 22;
-  const markerSize = hover ? hoverMarkerSize : standardMarkerSize;
+  const largeMarkerSize = 22;
+  let markerSize = standardMarkerSize;
+
+  // Enlarge marker if clicked or hovered
+  if (hover || clicked) {
+    markerSize = largeMarkerSize;
```

Comment:
change to single line like before
`const markerSize = (hover || clicked) ? largeMarkerSize : standardMarkerSize;`